src/building/suggested.md

```diff
@@ -3,6 +3,17 @@
 The full bootstrapping process takes quite a while. Here are some suggestions
 to make your life easier.
 
+## Installing a `pre-commit` hook
+
+The `rust` CI will automatically fail your build if it doesn't pass `tidy`, our
+internal tool for ensuring code quality. If you'd like, you can install a
+[git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)
+that will automatically run `tidy --bless` on each commit, to ensure
+your code is up to par. If you decide later that this behavior is
+undesirable, you can delete the `pre-commit` file in `.git/hooks`.
+
+A prebuilt git hook lives at [`src/etc/pre-commit.sh`](https://github.com/rust-lang/rust/blob/master/src/etc/pre-commit.sh) which can be copied directly into your `.git/hooks` folder.
```

Comment:
It can't be copied directly, it has to be called `pre-commit`, not `pre-commit.sh`.

Reply:
ugh yeah thanks for reminding me